### ISSUE TITLE
GetUserPlaylists Issue

### ISSUE THREAD

**Issue by suryapolina99** [JMPerez/spotify-web-api-js#176]
GetUserPlaylists function returns playlists that aren't owned by the user as well as the ones owned by them. There should be a way to filter the playlists not owned by the user out

**Comment by adrianhak** [comment]
Correct me if I'm wrong but I don't think that's possible (at least not through the web api).
The endpoint only has two query parameters, limit & offset. 

https://developer.spotify.com/documentation/web-api/reference/playlists/get-list-users-playlists/

**Comment by JMPerez** [comment]
@suryapolina99 The endpoint returns the "list of the playlists owned or followed by a Spotify user." That means it's totally possible to fetch playlists not owned by that user, but followed.

This library applies the same logic and doesn't add any extra processing of the results. If you would like to filter out those playlists not owned by the owner, you can check the `owner` field and leave those out.
In short, this endpoint removes the playlists from a user's library regardless of who is the owner.

The library doesn't perform extra logi
